### ISSUE TITLE
Link translations

### ISSUE THREAD

**Issue by vabatta** [kazupon/vue-i18n#26]
Hi,

I would suggest to add a linking function between the translations, like [angular-translate](http://angular-translate.github.io/docs/#/guide/02_getting-started) with the following syntax:
```js
{
  "SOME_NAMESPACE": {
    "OK_TEXT": "OK"
  },
  "ANOTHER_NAMESPACE": {
    "OK_TEXT": "@:SOME_NAMESPACE.OK_TEXT"
  }
}
```

Thanks!

**Comment by websanova** [comment]
Just started using this plugin, but I've used this feature with Angular as well and it's quite useful.

+1

**Comment by kazupon** [comment]
released v4.4.0
https://github.com/kazupon/vue-i18n/releases/tag/v4.4.0
